mappings/net/minecraft/level/chunk/Chunk.mapping

```diff
@@ -19,25 +21,93 @@ CLASS net/minecraft/class_43 net/minecraft/level/chunk/Chunk
 		ARG 2 tiles
 		ARG 3 x
 		ARG 4 z
+	METHOD method_858 equals (II)Z
+		ARG 1 x
+			COMMENT The chunk's x coordinate
+		ARG 2 z
+			COMMENT The chunk's z coordinate
 	METHOD method_859 getTileId (III)I
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
+	METHOD method_860 setTile (IIII)Z
+		ARG 1 x
+			COMMENT The relative X coordinate
+		ARG 2 y
+			COMMENT The relative Y coordinate
+		ARG 3 z
+			COMMENT The relative Z coordinate
```

Comment:
mojang uses the terminologoy "local x/y/z" for x y and z within a chunk, and I have been usign this terminology as well. Since it's standard, the javadocs may as well use it. 
However "relative" probably describes it a bit better, so perhaps call it relative/local or something idk